apps/balancer-tools/src/app/apr/(utils)/tokenApr.ts

```diff
@@ -199,9 +199,8 @@ async function getIntervalRates(
 ) {
   const [blockStart, blockEnd] = await Promise.all([
     await DefiLlamaAPI.findBlockNumber(chainName, timeStart),
-    await DefiLlamaAPI.findBlockNumber(chainName, timeEnd + SECONDS_IN_DAY),
+    await DefiLlamaAPI.findBlockNumber(chainName, timeEnd),
```

Comment:
@ribeirojose we were getting https://bleu.sentry.io/issues/4551833298/?project=4504990443700224&query= because we were searching one day ahead. Therefore, for the current date, it was searching on the future which returned 502 from coin.llama API

Reply:
good catch!